airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -471,8 +473,9 @@ def wait_for_job(self, url: str) -> str:
                     if not error_message:
                         # not all failed response can have "errorMessage" and we need to show full response body
                         error_message = job_info
-                    self.logger.error(f"JobStatus: {job_status}, sobject options: {self.sobject_options}, error message: '{error_message}'")
-
+                    self.logger.error(f"Job: {self.name}/{job_id}, JobStatus: {job_status}, sobject options: {self.sobject_options}, error message: '{error_message}'")
+                else:
+                    self.logger.info(f"Job: {self.name}/{job_id}, JobStatus: {job_status}, sobject options: {self.sobject_options}, error message: '{error_message}'")
```

Comment:
I fear `error_message` might not be defined here which would lead to `NameError: name 'error_message' is not defined`